apps/notifications/webpack.config.js

```diff
@@ -102,6 +103,7 @@ function getWebpackConfig(
 			new ExtensiveLodashReplacementPlugin(),
 			BuildMetaPlugin( { outputPath } ),
 		].filter( Boolean ),
+		devtool: isDevelopment ? 'inline-cheap-source-map' : 'source-map',
```

Comment:
A couple of Calypso apps use `source-map` as the `devtool` on production, but the Calypso client uses `hidden-source-map`. I think from the perspective of devex, `source-map` is better, while from a perf. perspective, one could argue `hidden-source-maps` saves some bytes. I decided to leave it as `source-map`, but happy to hear other people's thoughts about it, if any!

Reply:
Another point is that you get warnings in the browser console when a file includes a link to a source map, but one isn't served from the same location. If it doesn't make a difference to Sentry, we could just use hidden source map for all the Calypso apps